src/zck.cpp

```diff
@@ -596,11 +594,6 @@ namespace powerloader
         while (true)
         {
             size_t read = zck_read(zck, buf.data(), BUF_SIZE);
```

Comment:
I think we should change this to `ssize_t`: https://github.com/zchunk/zchunk/blob/dc937c3432b6bc1554de07eabb9222d2e278e6f4/include/zck.h.in#L88 and keep the error handling.